awscli/examples/logs/put-delivery-destination.rst

```diff
@@ -0,0 +1,22 @@
+**To Create or update a logical delivery destination**
+
+The following ``put-delivery-destination`` example creates a logical delivery destination named ``demo-delivery-destination``. ::
+
+    aws logs put-delivery-destination \
+        --name demo-delivery-destination \
+        "destinationResourceArn=arn:aws:s3:::code-whisperer-s3-bucket-demo"
```

Comment:
```suggestion
        --delivery-destination-configuration 'destinationResourceArn=arn:aws:s3:::code-whisperer-s3-bucket-demo'
```
This line appears to be missing a command line option. I believe it's supposed to be `--delivery-destination-configuration`, confirm whether or not this is correct and fix accordingly.